src/components/CurrentTime.js

```diff
@@ -0,0 +1,18 @@
+import {Text} from 'react-native';
+import * as React from 'react';
+
+const CurrentTime = () => {
+  const currentTime = new Date();
+
+  return (
+    <Text>
+      Time:{' '}
+      {currentTime.toLocaleString('en-us', {
+        hour: 'numeric',
+        minute: '2-digit',
+      })}
+    </Text>
+  );
+};
```

Comment:
If we wanted, since it's only referenced once, we can rewrite this as:
```
export const CurrentTime = () => (
  <Text>
      Time:{' '}
      {new Date().toLocaleString('en-us', {
        hour: 'numeric',
        minute: '2-digit',
      })}
  </Text>
);
```